imports/plugins/included/shipping-rates/server/hooks/hooks.js

```diff
@@ -34,6 +34,23 @@ function getShippingRates(previousQueryResults, cart) {
       return previousQueryResults;
     }
   }
+  if (!(cart.shipping && cart.shipping[0] && cart.shipping[0].address)) {
```

Comment:
One of our goals was to eliminate these hard-coded 0th indexes - I'm a bit concerned about introducing these here as it seems like adding technical debt immediately